src/plugins/controls/server/options_list/options_list_suggestions_route.ts

```diff
@@ -137,14 +133,13 @@ export const setupOptionsListSuggestionsRoute = (
     /**
      * Parse ES response into Options List Response
      */
-    const totalCardinality = get(rawEsResult, 'aggregations.unique_terms.value');
-    const suggestions = suggestionBuilder.parse(rawEsResult);
+    const results = suggestionBuilder.parse(rawEsResult, request);
```

Comment:
This is a bit awkward, but the best example of why this is necessary is the IP query - when parsing, we now need to know the size of the request because we are technically grabbing **double** what is necessary just in case the field contains a mix of both ipv4 and ipv6. We are trimming the list to `size` as part of the parse, so not sure if there's a better way of handling this? :shrug:

I also caught a bug where we weren't changing the sort of the IP query properly - woops! That's also fixed by sending in the `request` to the parse, since sorting has to be done after the ipv4 and ipv6 buckets are combined.